components/metadata-service/internal/externalapi/metadatahandler.go

```diff
@@ -33,14 +33,14 @@ func (mh *metadataHandler) CreateService(w http.ResponseWriter, r *http.Request)
 
 	serviceDefinition, apperr := mh.prepareServiceDefinition(r.Body)
 	if apperr != nil {
-		contextLogger.Errorf("Error creating new service: %s.", apperr.Error())
+		contextLogger.Errorf("metadata handler: preparing new service failed, %s", apperr.Error())
```

Comment:
According to Go conventions error messages should start with lowercase letter. The motivation is that if you concatenate a number of error messages the resulting string have correct casing. But in this case the above error message is returned to the user and written to the logs, so it should start from upper case letter.

Reply:
Ok, I'll take care of it.